data/locale/en-US.ini

```diff
@@ -55,18 +57,23 @@ Codec.H264.Profile.baseline="Baseline"
 Codec.H264.Profile.main="Main"
 Codec.H264.Profile.high="High"
 Codec.H264.Profile.high444p="High 4:4:4 Predictive"
+Codec.H264.Profile.Description="H.264 profile determines which features of the codec can be used.\nSince all modern devices can decode High perfectly, you should avoid using Main and Baseline,\nunless you need playback on very old devices and software decoding performance is not satisfactory.\nHigh 4:4:4 Predictive is required for YUV 4:4:4 color space."
 Codec.H264.Level="Level"
+Codec.H264.Level.Description="Level determines the upper limits of resolution, frame rate and bitrate for the video. Automatic will use the lowest level possible for your settings."
```

Comment:
The last statement is unconfirmed, and thus should be removed. There are encoders that willingly pick higher levels in order to give themselves more room or enable features that can better compress things.